cloud/storage/core/libs/features/features_config.cpp

```diff
@@ -17,34 +17,25 @@ bool ProbabilityMatch(double p, const TString& s)
 
 ////////////////////////////////////////////////////////////////////////////////
 
-TFeaturesConfig::TFeaturesConfig(
-        NProto::TFeaturesConfig config)
-    : Config(std::move(config))
+TFeaturesConfig::TFeatureInfo::TFeatureInfo(NProto::TFeatureConfig config)
+    : Whitelist(config.GetWhitelist())
+    , Blacklist(config.GetBlacklist())
+    , Value(config.GetValue())
 {
-    for (const auto& feature: Config.GetFeatures()) {
-        TFeatureInfo info;
-        info.IsBlacklist = feature.HasBlacklist();
-        info.CloudProbability = feature.GetCloudProbability();
-        info.FolderProbability = feature.GetFolderProbability();
-
-        const auto& cloudList = feature.HasBlacklist()
-            ? feature.GetBlacklist() : feature.GetWhitelist();
-
-        for (const auto& cloudId: cloudList.GetCloudIds()) {
-            info.CloudIds.emplace(cloudId);
-        }
+    double defaultProbability = config.HasWhitelist() ? 0.0 : 1.0;
```

Comment:
This is included only for backwards compatibility. Specifically not to break the config that is currently deployed for some clusters: blacklist is set, but no probability is specified